components/raftstore-v2/src/worker/tablet.rs

```diff
@@ -41,6 +42,13 @@ pub enum Task<EK> {
     DirectDestroy { tablet: Either<EK, PathBuf> },
     /// Cleanup ssts.
     CleanupImportSst(Box<[SstMeta]>),
+    /// Flush memtable before split
+    ///
+    /// on_flush_finish is some iff the task is sent from leader
```

Comment:
```suggestion
    /// on_flush_finish can't be none if the task is sent from leader
```

Reply:
"iff" should be used here as it cannot be some when sent from follower.

Reply:
And you should add some check in follower？

Reply:
Runner have no other information to see if it's follower or leader. Some or None is enough.